proxmox/nodes/tasks/client.go

```diff
@@ -23,13 +23,27 @@ func (c *Client) ExpandPath(_ string) string {
 	panic("ExpandPath of tasks.Client must not be used. Use BuildPath instead.")
 }
 
+// BaseTaskPath builds base task path using Task ID.
+func (c *Client) BaseTaskPath(taskID string) (string, error) {
```

Comment:
this function probably doesn't need to be exported.

Reply:
done